translate/rus.json

```diff
@@ -33,8 +33,8 @@
 "audio":"звук",
 "auto paste":"авто вставка",
 "auto":"авто",
-"average moves":"средн. ходов",
-"average time":"среднее время",
+"average moves":"сред. кол-во ходов",
+"average time":"сред. время партии",
```

Comment:
Here I am not sure about the length of the translation, I can revert this if you want